javascript-functions/main.js

```diff
@@ -0,0 +1,35 @@
+function convertMinutesToSeconds(minutes) {
+  return minutes * 60;
+}
+
+console.log('value of convertMinutesToSeconds(5):', convertMinutesToSeconds(5));
+
+function greet(name) {
+  return 'Hey,', name;
+}
```

Comment:
This isn't actually working correctly. If you take a look at your console output, you can see you are only getting the value of `name`. You aren't getting the `'Hey',`. You need to use string concatenation here